docs/features/graph view.md

```diff
@@ -49,6 +49,7 @@ Component.Graph({
     opacityScale: 1,
     removeTags: [], // what tags to remove from the graph
     showTags: true, // whether to show tags in the graph
+    enableRadial: true, // Whether to constrain the graph, similar to Obsidian
```

Comment:
Should this be included in the local graph as well? (with default set to `false`)